CONTRIBUTING.md

```diff
@@ -22,12 +22,25 @@ And run the test suite (should be all green with pending tests):
 
 If you have problems installing nmatrix, please consult the [nmatrix installation wiki](https://github.com/SciRuby/nmatrix/wiki/Installation) or the [mailing list](https://groups.google.com/forum/#!forum/sciruby-dev).
 
+**NOTE**: We use Travis CI for automatic testing for Daru. Since we have 
```

Comment:
Statement is wrong, please read about travis. daru is confirgured not travis.

Reply:
You can replace this paragraph with this -- 

`daru` is compatible with Ruby < 2.5, for later Ruby versions it may break - issue https://github.com/SciRuby/daru/issues/500, https://github.com/SciRuby/daru/issues/503

WDYT?

Reply:
Hey! Terribly sorry for letting this go undone for so late, was caught up with exams.  I did read up about Travis, but it is a lot of new terminologies so I think I got confused. I went ahead with what you have suggested, please take a look.